### PR TITLE
Feature/neighbors

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
#58 
#### What does this implement/fix? Explain your changes.
Add support for sci-kit learn Neighbors' ML Models

### CODE REVIEW

**Comment by AHReccese** [comment]
@sadrasabouri 
Do not merge after reviewing the PR, I will take care of the rest.

File: pymilo/transporters/neighbors_tree_transporter.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    def deserialize(self, data, key, model_type):
+        """
+        Deserialize the special _tree field of the Neighbors model.
+
+        The associated tree_ field of the pymilo serialized model, is extracted through
+        it's previously serialized parameters.
+        deserialize the data[key] of the given model which type is model_type.
+        basically in order to fully deserialize a model, we should traverse over all the keys of its serialized data dictionary and
+        pass it through the chain of associated transporters to get fully deserialized.
```
Comment (sadrasabouri) [inline pymilo/transporters/neighbors_tree_transporter.py:49]: That's too much for doc-string. I think we you can simplify it into 1/2 sentence(s).

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 | Clustering &#x2705; | -  | 
 | Naïve Bayes &#x2705; | -  | 
 | Support vector machines (SVMs) &#x274C; | -  | 
-| Nearest Neighbors &#x274C; | -  | 
+| Neighbors &#x274C; | -  |
```
[the remark this answers is not in the document]
Comment (AHReccese) [reply README.md]: Sure Sure, It was a tiny conflict which is now resolved.

File: pymilo/transporters/neighbors_tree_transporter.py (hunk 1 above)
Comment (AHReccese) [reply pymilo/transporters/neighbors_tree_transporter.py]: Done.
I will also apply a deep refactoring in docstrings in a separate enhancing pr.
@sadrasabouri

**Review by sadrasabouri — APPROVED** [review]

**Comment by sadrasabouri** [comment]
Thanks for your efforts @AHReccese
I reviewed it and it's ready to merge. Feel free to merge it anytime.

**Pull request merged**